eks_infrastructure/patch_hosts.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#/ Usage: ./patch_hosts.sh
+
+set -ex
+source ./helper.sh
+
+# Parse parameters from build_param.json config file
+EKS_CLUSTERS=($(jq -r '.eks_clusters[]' build_param.json))
+CONTEXTS=($(jq -r '.contexts[]' build_param.json))
+EKS_VERSION=$(jq -r '.eks_version' build_param.json)
+CLUSTER_AUTOSCALAR_IMAGE_VERSION=$(jq -r '.cluster_autoscalar_image_version' build_param.json)
+
```

Comment:
Add validation for operation, if upgrade operation is not found, skip the job
Or
If we are creating a duplicate file similar to build_param.json, let's not include operation field, we can only keep cluster names and other parameters if required

Reply:
there is no operation variable in this file.